integration/basic_test.go

```diff
@@ -60,16 +60,8 @@ func TestFindNothing(t *testing.T) {
 	t.Parallel()
 	ctx, collection := setup.Setup(t)
 
-	cursor, err := collection.Find(ctx, bson.D{})
```

Comment:
This one is the same as query compat test

```go
"Empty": {
  filter: bson.D{},
},
```

and this is why it's removed, right?

Reply:
Yes I removed test for find because it's covered by compat.

I added comment in that test.